### PR TITLE
KR login view

### PR DESCRIPTION
**Pull request by kjreczek**
Zrobiłem widok do logowania, wykorzystane komponenty są pokryte testami w ramach innych zadań.

### CODE REVIEW

**Comment by fszwedo** [comment]
@kjreczek I see all the files are in UnitImplementation folder? shouldn't they be in src instead?

File: UnitImplementations/UIComponents/LogIn/LogIn.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+container.appendChild(title);
+document.body.appendChild(container);
+
+const input = generateTextInput('Nickname', 'name');
+container.appendChild(input);
+
+const button = generateRedButton('Log In', ()=>{});
+
+button.style.width = "200px";
+button.style.height = "50px";
```
Comment (robert-czwartosz) [inline UnitImplementations/UIComponents/LogIn/LogIn.ts:18]: I suggest to add to .css file selector .main-container > button{width: 200px; height: 50px}
and remove this code.

**Review by robert-czwartosz — COMMENTED** [review]
Component looks good, but there are some things to correct in code.

File: UnitImplementations/UIComponents/LogIn/LogIn.ts (hunk 1 above)
Comment (kjreczek) [reply UnitImplementations/UIComponents/LogIn/LogIn.ts:18]: Thanks, I changed it.

**Comment by kjreczek** [comment]
> @kjreczek I see all the files are in UnitImplementation folder? shouldn't they be in src instead?

@fszwedo I understood that: since there are no new reusable components, I can only prepare the view, I just put it to unitimplementation. It will be adjusted later anyway.

**Review by robert-czwartosz — APPROVED** [review]

**Review by fszwedo — APPROVED** [review]

**Pull request merged**